config/initializers/inflections.rb

```diff
@@ -17,4 +17,5 @@
 # hack for one acronym from KCSC, but need a better solution ...
 ActiveSupport::Inflector.inflections(:en) do |inflect|
   inflect.acronym 'ABI'
-end
\ No newline at end of file
+  inflect.irregular 'app_data', 'app_data'
```

Comment:
Looks kinda weird to me. Can you explain?

Reply:
It's a hack so that rails does not singularize this word/expression.